### DISCUSSION TITLE
Generate swagger file without starting service

### DISCUSSION THREAD

**Discussion by Hedgehogues** [fastapi/fastapi#7452] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I generate the swagger without starting the service?

**Comment by phy25** [comment, marked as answer]
Duplicate of #1490.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for closing the issue @Hedgehogues  :+1:
